### PR TITLE
API-23430/Slack dashboard URLs

### PR DESCRIPTION
**Pull request by ediiotero**
Story: [API-23430](https://vajira.max.gov/browse/API-23430)

Update Slack URLs to match changes on the portal to include the env in the URL.

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -104,7 +111,7 @@ const config = {
   LOAST_VERSION: 'unknown',
   LOAST_URL_PATTERN:
     process.env.LOAST_URL_PATTERN ||
-    'https://hub.lighthouse.va.gov/provider-dashboard/apis/{API_NAME}/versions/{VERSION}/loast',
+    `https://hub.lighthouse.va.gov/provider-dashboard/apis/{API_NAME}/${portalEnv}/versions/{VERSION}/loast`,
```
Comment (mheusgit) [inline src/config.ts:114]: The dashboard URLs were originally designed to closely match the actual request on DocServer side.  I prefer if the env path param proceeds the rest.

https://hub.lighthouse.va.gov/provider-dashboard/${portalEnv}/apis/{API_NAME}/versions/{VERSION}/loast

File: src/config.ts [hunk 2]
```diff
@@ -53,6 +58,13 @@ const ENV_METADATAS: { [env: string]: EnvironmentMetadata[] } = {
 };
 
 const env = process.env.OAS_ENV || 'unknown';
+
+let portalEnv = env;
+
+if (env === EnvType.DEV || env === 'unknown') {
+  portalEnv = PortalEnv.QA;
```
Comment (mheusgit) [inline src/config.ts:65]: The logic does not appear to support links to DocServer staging or prod

Comment (ediiotero) [reply src/config.ts:65]: I might be missing something but `portalEnv` is set to the current `env` that could be staging, dev, or prod. The `If` is only there to change it from dev to qa to be in line with the links on the portal.

Comment (mheusgit) [reply src/config.ts:65]: I misread it.  Your good then

**Review by mheusgit — DISMISSED** [review]
Looks good

**Review by dsteeve — APPROVED** [review]
I like the "dashboardEnvContext" change.  I think this aligns better with the future for it to be DEV.  

I realize the actual docserver context is QA (which is our DEV) -- because blue.{env}.lighthouse.va.gov doesn't have a DEV.  However the future will be LHDI which does have a DEV and a QA but that QA is more like STAGING.  LHDI won't have a STAGING, so thus I think DEV is a better choice here.  

LOL! I don't think any sane person can read that above paragraph.

**Comment by dsteeve** [comment]
> I like the "dashboardEnvContext" change. I think this aligns better with the future for it to be DEV.
> 
> I realize the actual docserver context is QA (which is our DEV) -- because blue.{env}.lighthouse.va.gov doesn't have a DEV. However the future will be LHDI which does have a DEV and a QA but that QA is more like STAGING. LHDI won't have a STAGING, so thus I think DEV is a better choice here.
> 
> LOL! I don't think any sane person can read that above paragraph.

Actually, think this value needs to remain as 'qa' for now -- and change that to 'dev' as a separate ticket.

**Review by dsteeve — CHANGES_REQUESTED** [review]
--QA is an env highly baked into the provider dashboard at  the moment, so will make that change to DEV as a separate effort.

**Review by dsteeve — APPROVED** [review]
looks good!

**Pull request merged**